packages/cli/src/commands/deploy.ts

```diff
@@ -1,21 +1,23 @@
 import { Command, flags } from '@oclif/command'
 import { deployToCloudProvider } from '../services/provider-service'
-import { compileProjectAndLoadConfig } from '../services/config-service'
+import {
+  cleanProductionSandbox,
+  compileProjectAndLoadConfig,
+  createProductionSandbox,
```

Comment:
I wonder if sandbox-related operations belong to this scope, they could perfectly be used from the `compileProjectAndLoadConfig` process without managing them from here, they're a bit "low level" to be interesting to the user 🤔

Reply:
I first wrote it in the `compileAndLoadConfig`, but I had quickly to change it because of the following reasons:
- Many commands use the `compileAndLoadConfig`, but only the `deploy` command needs to create a sandbox.
- That function works with a project, and we first need to create the project, which in this case is done with `createProductionSandbox`